starlette_admin/contrib/odmantic/admin.py

```diff
@@ -19,6 +19,7 @@ def __init__(
         route_name: str = "admin",
         logo_url: Optional[str] = None,
         login_logo_url: Optional[str] = None,
+        favicon_url: Optional[str] = None,
```

Comment:
It's better to move it down to ensure backward compatibility

Reply:
totally agree, I 'll update it.